### PR TITLE
Ensure to test SEARCH, PURGE and MKCALENDAR

### PR DESCRIPTION
**Pull request by Hywan**
These methods are defined in `http_parser.h` but not tested.

### CODE REVIEW

**Comment by Hywan** [comment]
ping?

File: http_parser.c [hunk 1]
```diff
@@ -965,9 +965,7 @@ size_t http_parser_execute (http_parser *parser,
           case 'M': parser->method = HTTP_MKCOL; /* or MOVE, MKACTIVITY, MERGE, M-SEARCH, MKCALENDAR */ break;
           case 'N': parser->method = HTTP_NOTIFY; break;
           case 'O': parser->method = HTTP_OPTIONS; break;
-          case 'P': parser->method = HTTP_POST;
-            /* or PROPFIND|PROPPATCH|PUT|PATCH|PURGE */
-            break;
+          case 'P': parser->method = HTTP_POST; /* or PROPFIND, PROPPATCH, PUT, PATCH, PURGE */ break;
```
Comment (indutny) [inline http_parser.c]: Please apply 80 column limit. I know it is different in other places of the file, but I'd like to enforce this style on all future changes.

Comment (indutny) [inline http_parser.c]: Actually, you may just want to leave it as it is ;)

**Comment by indutny** [comment]
LGTM, with only one nit. Thank you!

**Comment by indutny** [comment]
Landed in b5bcca8f928cafff36b8a0d04c2c3a771205cb62

**Comment by Hywan** [comment]
Thanks :-]!

**Pull request closed without merging**